### PR TITLE
PPC: Tweak video optimization language

### PR DESCRIPTION
**Pull request by dreamofabear**
## Context

Related to #7257.

## Summary

- Add a caption with some explanation on what/why videos should be "optimized"
- Remove the "video resolution too large" check (which triggered for 4K+ videos)

## Relevant Technical Choices

N/A

## User-facing changes

Before:
Screen Shot 2021-05-17 at 7 51 06 PM

After: 
Screen Shot 2021-05-17 at 7 54 14 PM

### QA

This PR can be tested by following these steps:

1. Same steps as #7257, just a slightly different visual treatment.

### UAT

- [x] UAT should use the same steps as above.

### Does this PR have a security-related impact?

No.

### Does this PR change what data or activity we track or use?

No.

### Does this PR have a legal-related impact?

No.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

See #7257

### CODE REVIEW

File: assets/src/edit-story/app/prepublish/components/videoOptimization.js [hunk 1]
```diff
@@ -62,6 +67,16 @@ export function VideoOptimization({ element }) {
 
   return (
     <Container>
+      <Caption>
+        <ul>
+          <li>
+            {__(
+              'Videos larger than 720p can cause slower loading and higher bandwidth costs.',
+              'web-stories'
+            )}
```
Comment (swissspidy) [inline assets/src/edit-story/app/prepublish/components/videoOptimization.js]: I recommend using `sprintf` here so the `720p` is not accidentally translated.

```js
sprintf(
  /* translators: %s: video resolution (720p) */
  __('Videos larger than %s can cause slower loading and higher bandwidth costs.', 'web-stories'),
  '720p'
)
```

File: assets/src/edit-story/app/prepublish/guidance/media.js [hunk 2]
```diff
@@ -58,18 +56,6 @@ function videoElementResolution(element) {
...
-  const videoResolutionHigh =
-    element.resource?.sizes?.full?.height >= MAX_VIDEO_HEIGHT &&
-    element.resource?.sizes?.full?.width >= MAX_VIDEO_WIDTH;
-
-  if (videoResolutionHigh) {
-    return {
-      type: PRE_PUBLISH_MESSAGE_TYPES.GUIDANCE,
-      elementId: element.id,
-      message: MESSAGES.MEDIA.VIDEO_RESOLUTION_TOO_HIGH.MAIN_TEXT,
-      help: MESSAGES.MEDIA.VIDEO_RESOLUTION_TOO_HIGH.HELPER_TEXT,
```
Comment (swissspidy) [inline assets/src/edit-story/app/prepublish/guidance/media.js:70]: Can remove these constants too from `constants.js`

File: assets/src/edit-story/app/prepublish/constants.js [hunk 3]
```diff
@@ -638,7 +638,7 @@ export const MESSAGES = {
       ),
     },
     VIDEO_NOT_OPTIMIZED: {
-      MAIN_TEXT: __('Video not optimized', 'web-stories'),
+      MAIN_TEXT: __('Optimize video size', 'web-stories'),
```
Comment (BrittanyIRL) [inline assets/src/edit-story/app/prepublish/constants.js]: This update will require some unit tests also get updated here: `edit-story/app/prepublish/guidance/test/media`

**Comment by dreamofabear** [comment]
Thanks for the reviews!

File: assets/src/edit-story/app/prepublish/constants.js [hunk 4]
```diff
@@ -620,25 +619,8 @@ export const MESSAGES = {
...
-              "Optimize reach and engagement by accounting for the large number of mobile devices don't support video resolutions larger than 4K",
-              'web-stories'
-            )}
-          </li>
-        </ul>
-      ),
-    },
     VIDEO_NOT_OPTIMIZED: {
-      MAIN_TEXT: __('Video not optimized', 'web-stories'),
+      MAIN_TEXT: __('Optimize video size', 'web-stories'),
```
Comment (BrittanyIRL) [inline assets/src/edit-story/app/prepublish/constants.js:623]: i think the way you adjusted this is fine, but the text you added in the component could have also lived here as `HELPER_TEXT` to keep with the reusable pattern. Just a heads up for next time. Sorry I didn't catch it earlier!

File: assets/src/edit-story/app/prepublish/guidance/media.js [hunk 5]
```diff
@@ -169,7 +155,12 @@ export function videoElementOptimized(element = {}) {
       type: PRE_PUBLISH_MESSAGE_TYPES.GUIDANCE,
       elementId: element.id,
       message: MESSAGES.MEDIA.VIDEO_NOT_OPTIMIZED.MAIN_TEXT,
-      help: <VideoOptimization element={element} />,
+      help: (
+        <VideoOptimization
+          element={element}
+          caption={MESSAGES.MEDIA.VIDEO_NOT_OPTIMIZED.HELPER_TEXT}
```
Comment (BrittanyIRL) [inline assets/src/edit-story/app/prepublish/guidance/media.js:161]: <3 thank you!

File: assets/src/edit-story/app/prepublish/constants.js (hunk 4 above)
Comment (dreamofabear) [reply assets/src/edit-story/app/prepublish/constants.js:623]: Good idea, done.

**Review by BrittanyIRL — APPROVED** [review]

**Pull request merged**